### PR TITLE
[TE] rootcause poc - controller cleanup and jsdoc

### PR DESCRIPTION
**Pull request by apucher**

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const ROOTCAUSE_TAB_DIMENSIONS = "dimensions";
 const ROOTCAUSE_TAB_METRICS = "metrics";
 const ROOTCAUSE_TAB_EVENTS = "events";
 
+const ROOTCAUSE_SERVICE_ROUTE = "route";
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: Please use single quotes `'` for string in JS. (This is a convention we adopted) 
Applies to all prs.

File: thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js [hunk 2]
```diff
@@ -34,6 +33,8 @@ export default Ember.Controller.extend({
   //
   // services
   //
+  authService: Ember.inject.service('session'),
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: any reason why we changed the name here? (curious)

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: Naming collision. 'session' currently refers to 'RCA session' in the route and controller - this has to be refactored in the future

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: fixed

File: thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js (hunk 1 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: fixed

**Pull request merged**